jetty-ee9/jetty-ee9-webapp/src/main/java/org/eclipse/jetty/ee9/webapp/AbstractConfiguration.java

```diff
@@ -19,6 +19,8 @@
 import java.util.List;
 import java.util.stream.Collectors;
 
+import org.eclipse.jetty.util.ClassMatcher;
```

Comment:
This changes the signatures of the methods in this class that take or return ClassMatcher.